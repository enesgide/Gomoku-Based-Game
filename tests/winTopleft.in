place A1
place A2
place B2
place B3
place C3
place C4
place D4
place D5
place E5