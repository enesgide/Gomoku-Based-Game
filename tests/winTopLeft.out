Black wins!
A1A2B2B3C3C4D4D5E5
Thank you for playing!
